src/components/BrowserApp.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BrowserRouter from 'react-router-dom/BrowserRouter';
+import PlatformApp from '../components/PlatformApp';
+
+/**
+ * A simple component to wrap the base PlatformApp with the BrowserRouter
+ */
+class BrowserApp extends React.Component {
+	render() {
+		const { basename, store, routes } = this.props;
+		return (
+			<BrowserRouter basename={basename}>
+				<PlatformApp store={store} routes={routes} />
+			</BrowserRouter>
+		);
+	}
+}
+
+BrowserApp.propTypes = {
+	routes: PropTypes.array.isRequired,
+	store: PropTypes.object.isRequired,
+	baseUrl: PropTypes.string.isRequired,
```

Comment:
ah, thanks, I was flipping back and forth on the name of this property because some packages refer to it as baseUrl and some refer to it as basename so I'm undecided where I should make the name switch in the platform interface...